lib/diagnostic.rb

```diff
@@ -46,6 +46,13 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    fh = open filename
+    # need to read each line
+    fh.each do |line|
+      # need to replace newline character with a space
+      line.replace(' ')
+      # need to concatenate and return
+      line.join
```

Comment:
`.join` is a method on Arrays. Is line an array? 🤔 
https://ruby-doc.org/core-2.2.0/Array.html#method-i-join